File: CYBaseProject/CYBaseProject/Code/Utils/UI/UILabel+CYUtils.m
//
//  UILabel+EUUtils.m
//  Euaaaios
//
//  Created by juanjuanHuang on 16/4/10.
//  Copyright © 2016年 euaaa. All rights reserved.
//

#import "UILabel+CYUtils.h"

@implementation UILabel (CYUtils)

+ (UILabel *)labelWithFont:(UIFont *)font textColor:(UIColor *)textColor {
    
    UILabel *label = [[UILabel alloc] init];
    label.font = font;
    label.textColor = textColor;
    label.backgroundColor = [UIColor clearColor];
    return label;
}

@end

File: CYBaseProjectSwift/CYBaseProjectSwift/Code/Utils/UI/UILabel+CYUtils.swift
//
//  UILabel+CYUtils.swift
//  CYBaseProjectSwift
//
//  Created by HuangQiSheng on 4/22/16.
//  Copyright © 2016 Jasper. All rights reserved.
//

import UIKit

extension UILabel {

    convenience init(font: UIFont, textColor: UIColor) {
        
        self.init(frame: CGRectZero)
        self.font = font
        self.textColor = textColor
    }
    
}
